phd-tests/framework/src/artifacts/store.rs

```diff
@@ -311,6 +373,88 @@ impl Store {
             )),
         }
     }
+
+    pub fn get_crucible_downstairs(&self) -> anyhow::Result<Utf8PathBuf> {
```

Comment:
Thanks for refactoring all this stuff--the store looks a lot cleaner with all these bits pulled out into proper functions!